Detectors/ITSMFT/ITS/reconstruction/src/CookedTracker.cxx

```diff
@@ -164,7 +149,7 @@ void CookedTracker::setExternalIndices(CookedTrack& t) const
   Int_t noc = t.getNumberOfClusters();
   for (Int_t i = 0; i < noc; i++) {
     Int_t index = t.getClusterIndex(i);
-    Cluster *c = getCluster(index);
+    const Cluster *c = getCluster(index);
     Int_t idx=c->GetUniqueID();
```

Comment:
see comment on SetUniqueID in clusterer

Reply:
Yes, sure.  There must be some  better solution to this...